mockup/patterns/relateditems/pattern.js

```diff
@@ -98,11 +98,11 @@ define([
   'mockup-utils',
   'pat-registry',
   'translate',
-  'text!mockup-patterns-relateditems-url/templates/breadcrumb.xml',
-  'text!mockup-patterns-relateditems-url/templates/favorite.xml',
-  'text!mockup-patterns-relateditems-url/templates/result.xml',
-  'text!mockup-patterns-relateditems-url/templates/selection.xml',
-  'text!mockup-patterns-relateditems-url/templates/toolbar.xml',
+//  'text!mockup-patterns-relateditems-url/templates/breadcrumb.xml',
+//  'text!mockup-patterns-relateditems-url/templates/favorite.xml',
+//  'text!mockup-patterns-relateditems-url/templates/result.xml',
+//  'text!mockup-patterns-relateditems-url/templates/selection.xml',
+//  'text!mockup-patterns-relateditems-url/templates/toolbar.xml',
```

Comment:
@b4oshany These have probably been forgotten here and are cause for the test errors. But why did you need to disable these imports in your setup in the first place?